python/pyarrow/includes/chrono.pxd

```diff
@@ -0,0 +1,18 @@
+from libc.stdint cimport *
+
+
+cdef extern from "<chrono>" namespace "std::chrono":
+    cdef cppclass duration:
```

Comment:
I'm curious why  the template parameter isn't needed here, but I guess it's inferred from the constructor parameter type?